src/util.h

```diff
@@ -31,6 +31,7 @@
 static const bool DEFAULT_LOGTIMEMICROS = false;
 static const bool DEFAULT_LOGIPS        = false;
 static const bool DEFAULT_LOGTIMESTAMPS = true;
+static const int64_t DEFAULT_MAX_TIME_ADJUSTMENT = 70 * 60;
```

Comment:
Yep, sure thing, I'll update that. I had thought of that and just figured I'd leave it with some other defaults since then I didn't have to add the include in init.cpp.